includes/Plugin.php

```diff
@@ -207,6 +217,7 @@ class Plugin {
 	public function register() {
```

Comment:
I would the opportunity to work on https://github.com/google/web-stories-wp/pull/5467 

This register method is overly long now. The problem with letting this go out, it means public context `$this->web_stories_block`